front/app/components/FollowUnfollow/index.tsx

```diff
@@ -41,14 +42,23 @@ const FollowUnfollow = ({
   const { mutate: addFollower, isLoading: isAddingFollower } = useAddFollower();
   const { mutate: deleteFollower, isLoading: isDeletingFollower } =
     useDeleteFollower();
+  const { treatment, send } = useABTest({
+    experiment: 'Following an idea text',
```

Comment:
```suggestion
    experiment: `Following an idea text (${locale})`,
```

I think it makes sense to translate the treatments, since what works well in one language may not work well in another language. However maybe it is useful to add the locale to the experiment name and see them as separate experiments, since that is probably how you want to analyse them. i.e. you will want to see for each language how translation A compares to translation B, not how all translations of all languages compare to each other

Reply:
Good point 💪